modules/rosa-getting-started-verify-aws-quota.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-verify-aws-quota_{context}"]
+= Verifying AWS quota availability
+
+Verify that the required resource quotas are available for your account in the default AWS region.
+
+.Prerequisites
+
+* You have an AWS account.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+
+.Procedure
+
+. Verify if the required resource quotas are available in your default region:
```

Comment:
"Verify that..."

Reply:
I will keep this open until the discussion in the preceding thread is complete.

Reply:
Based on our subsequent conversation in Slack, I will leave the line as "verify if".